### PR TITLE
Reword LSP tracker plugin description

### PR DESCRIPTION
**Pull request by fcollonval**
## References

Back ticks in plugin description are breaking the doc generation; see https://github.com/jupyterlab/jupyterlab/actions/runs/6388616044/job/17338748652.

## Code changes

Remove back ticks.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

File: packages/lsp-extension/src/index.ts [hunk 1]
```diff
@@ -310,7 +310,7 @@ function addRunningSessionManager(
 
 const adapterTrackerPlugin: JupyterFrontEndPlugin<IWidgetLSPAdapterTracker> = {
   id: '@jupyterlab/lsp-extension:tracker',
-  description: 'Provides the `WidgetLSPAdapter`s tracker.',
+  description: "Provides the WidgetLSPAdapter's tracker.",
```
Comment (krassowski) [inline packages/lsp-extension/src/index.ts:313]: No strong opinion on this one, but if this gets pulled into rst, would a double backtick work?

```suggestion
  description: 'Provides the ``WidgetLSPAdapter``s tracker.',
```

Comment (fcollonval) [reply packages/lsp-extension/src/index.ts:313]: Let's try :wink:

Comment (krassowski) [reply packages/lsp-extension/src/index.ts]: It seems it does not:

```
/home/runner/work/jupyterlab/jupyterlab/docs/source/extension/plugins_list.rst:118: WARNING: Inline literal start-string without end-string.
```

**Comment by fcollonval** [comment]
Sphinx is not happy :cry: 

```
/home/runner/work/jupyterlab/jupyterlab/docs/source/extension/plugins_list.rst:118: WARNING: Inline literal start-string without end-string.
```

File: galata/test/documentation/plugins.test.ts-snapshots/plugins-documentation-linux.json [hunk 2]
```diff
@@ -116,7 +116,7 @@
   "@jupyterlab/lsp-extension:feature": "Provides the language server feature manager.",
   "@jupyterlab/lsp-extension:settings": "Provides the language server settings.",
   "@jupyterlab/lsp:ILSPCodeExtractorsManager": "Provides the code extractor manager.",
-  "@jupyterlab/lsp-extension:tracker": "Provides the WidgetLSPAdapter's tracker.",
+  "@jupyterlab/lsp-extension:tracker": "Provides the ``WidgetLSPAdapter``s tracker.",
```
Comment (krassowski) [inline galata/test/documentation/plugins.test.ts-snapshots/plugins-documentation-linux.json:119]:
```suggestion
  "@jupyterlab/lsp-extension:tracker": "Provides the tracker of `WidgetLSPAdapter`.",
```

File: packages/lsp-extension/src/index.ts [hunk 3]
```diff
@@ -310,7 +310,7 @@ function addRunningSessionManager(
 
 const adapterTrackerPlugin: JupyterFrontEndPlugin<IWidgetLSPAdapterTracker> = {
   id: '@jupyterlab/lsp-extension:tracker',
-  description: 'Provides the `WidgetLSPAdapter`s tracker.',
+  description: 'Provides the ``WidgetLSPAdapter``s tracker.',
```
Comment (krassowski) [inline packages/lsp-extension/src/index.ts:313]:
```suggestion
  description: 'Provides the tracker of `WidgetLSPAdapter`.',
```

**Review by krassowski — COMMENTED** [review]
I think we've been fooled. Maybe it is not the backtick which is problematic. Maybe it is the `s` following the backtick that matters.

**Review by krassowski — APPROVED** [review]
We are green now:

image

It also looks fine on the doc preview:

image


The remaining failures are unrelated.

**Pull request merged**